scripts/sommelier

```diff
@@ -183,9 +183,19 @@ start_app() {
     . "$SNAP/sommelier/hooks/pre-start"
   fi
 
+  if [[ -e "${ACTION_ARGS}" ]]; then
```

Comment:
Can you put this `if` inside of the `for` loop, so we check each argument individually? Then if the app has multiple arguments, we only run `winepath -w` for the arguments which are valid paths.